corehq/apps/enterprise/dispatcher.py

```diff
@@ -0,0 +1,13 @@
+from django.utils.decorators import method_decorator
+
+from corehq.apps.reports.dispatcher import ReportDispatcher
+from corehq.apps.enterprise.decorators import require_enterprise_admin
+
+
+class EnterpriseInterfaceDispatcher(ReportDispatcher):
```

Comment:
What is an 'Enterprise Interface'? Other dispatchers I'm seeing are: `ProjectReportDispatcher`, `CustomProjectReportDispatcher`, `DomainReportDispatcher`, etc, which all seem fairly straight-forward. I'm sure there's a better name here for what this class is responsible for.